testsuite/features/support/constants.rb

```diff
@@ -220,7 +220,8 @@
 # Then take a look at the Parent Channel selections
 BASE_CHANNEL_BY_CLIENT = {
   'SUSE Manager' => {
-    'proxy' => 'SLE-Product-SUSE-Manager-Proxy-4.3-Pool for x86_64',
+    'proxy_container' => 'SLE-Micro-5.5-Pool for x86_64',
```

Comment:
I would remove the rest of changes and have only this minor change in Head/Master.

```suggestion
    'proxy' => 'SLE-Micro-5.5-Pool for x86_64',
```

Keeping Manager-4.3 untouch.

Reply:
This needs to stay because when we deploy e.g. HEAD, it will look into this array. To keep 4.3 untouched, I will not back port this PR.